src/test.ts

```diff
@@ -1,7 +0,0 @@
-class MyClass {
-  private _myVar = 'hello';
-
-  foo() {
-    console.log(this._myVar)
-  }
-}
```

Comment:
Whats this about?

Reply:
It was some leftover testing code that leaked through